python-regression/DB_References.txt

```diff
@@ -0,0 +1,54 @@
+Test DB's 
+All test db's are stored in an Iota Foundation s3 bucket:
+[ https://s3.eu-central-1.amazonaws.com/iotaledger-dbfiles/dev/* ]
+
+Blowball Tests - Machine 3
+Name: Blowball_Tests_db.tar
+
+Description: This db was created by bootstrapping the testnet coo tool to create a stream of milestones. As these milestones were being issued, the node was then spammed with valid and invalid transactions. Once a few invalid transactions were stored in the db, a side tangle referencing one of these transactions was generated, forming a chain of transactions only referencing previous side-tangle transactions, thus separating it from the original flow of transactions. This DB contains a few thousand valid, invalid and side tangle transactions to traverse through for the gtta tests. 
+
+Scripts Used: 
+-https://github.com/iotaledger/tangle-tools/tree/python-tools/python-tools: 
+    invalid_transaction_spammer.py, 
```

Comment:
since this is a private repository please create a specialized folder for this repo and add the scripts here